defaults/main.yml

```diff
@@ -5,20 +5,26 @@
 consul_debug: no
 
 ### Package
-consul_version: "{{ lookup('env','CONSUL_VERSION') | default('0.8.2', true) }}"
+consul_version: "{{ lookup('env','CONSUL_VERSION') | default('0.8.3', true) }}"
```

Comment:
The last commit on the upstream repo updates the project to support Consul 0.8.3, but the default installation still downloads 0.8.2. This is an unrelated bug fix, but will probably just disappear if fixed upstream, correct?